mobile/lib/services/entity_service.dart

```diff
@@ -219,4 +220,22 @@ class EntityService {
       rethrow;
     }
   }
+
+  Future<String> getHashForIds(List<String> personIds) async {
+    String preHash = "";
+
+    for (final id in personIds) {
+      final entity = await getEntity(EntityType.cgroup, id);
+
+      if (entity != null) {
+        preHash = "$id:${entity.updatedAt}_";
+      }
```

Comment:
We can just read list of updatedAt or better just max updatedAt for the list of personIDs from db and use that to identify if recomputing is required?

Reply:
- No, we are creating hash not only based on updatedAt, but whose updatedAt it is.
- It might be that one person is removed from selection but we still have same hash if we use the max updatedAt concept.
- If we readList of updatedAt (I did that before), then it might be possible two person have same updatedAt and hash remains the same (I assume).
- This is best way as it not only specifies the updatedAt but whose updatedAt it is, so hash is always unique.